vllm-proxy/src/app/api/v1/openai.py

```diff
@@ -158,7 +176,7 @@ def strip_empty_tool_calls(payload: dict) -> dict:
     filtered_messages = []
     for message in payload["messages"]:
         # If the message has tool_calls, filter out empty ones
-        if "tool_calls" in message and len(message["tool_calls"]) == 0:
+        if "tool_calls" in message and message["tool_calls"] is not None and len(message["tool_calls"]) == 0:
```

Comment:
```suggestion
        if "tool_calls" in message and isinstance(message["tool_calls"], list) and  len(message["tool_calls"]) == 0:
```

Seems like this is better. But please test. I didn't test it.

Reply:
Sure, I think this also reduces other potential values, it's more robust, will fix